yaml-to-php.php

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+use function Termwind\{render};
+
+require __DIR__ . '/vendor/autoload.php';
+
+$exit = 0;
+
+/**
+ * @return Generator<string>
+ */
+function getDirectories(): Generator
+{
+    $finder = new Finder();
+    $finder
+        ->in(__DIR__)
+        ->directories()
+        ->exclude('vendor')
+        ->depth(3);
+
+    foreach ($finder as $directory) {
+        if ('config' === $directory->getFilename()) {
+            yield $directory->getPathname();
+        }
+    }
+}
+
+render(<<<HTML
+    <div class="my-1 mx-2 bg-blue p-1 px-2">
+        <div class="font-bold">Convert configs</div>
+    </div>
+HTML);
+
+$successes = [];
+$errors = [];
+
+foreach (getDirectories() as $directory) {
+    $process = new Process([
+        \PHP_BINARY,
+        __DIR__ . '/vendor/bin/config-transformer',
+        'switch-format',
+        $directory,
+    ]);
+
+    $relativeDirectory = str_replace(__DIR__ . '/', '', $directory);
+
+    $class = 'text-green';
+    $status = 'success';
+
+    try {
+        $process->mustRun();
+
+        $successes[$relativeDirectory] = $process->getOutput();
+    } catch (ProcessFailedException $exception) {
+        $exit = 1;
+        $errors[$relativeDirectory] = $exception->getMessage();
+
+        $class = 'text-red';
+        $status = 'failed';
+    }
+
+    render(<<<HTML
+    <div class="flex space-x-1 mx-2">
+        <span class="font-bold">{$relativeDirectory}</span>
+        <span class="flex-1 content-repeat-[.] text-gray"></span>
+        <span class="font-bold {$class} uppercase">{$status}</span>
+    </div>
+HTML);
+}
+
+$process = new Process([
+    \PHP_BINARY,
+    __DIR__ . '/vendor/bin/ecs',
+    '--fix',
+    '--no-progress-bar',
+]);
+
+render(<<<HTML
+    <div class="my-1 mx-2 bg-blue text-white p-1 px-2">
+        <div class="font-bold">Fix code style</div>
+    </div>
+HTML);
+
+if (\in_array($process->run(), [0, 1], true)
+    // TODO currently 2 runs of ecs is required as "StandaloneLineInMultilineArrayFixer" is only applied correctly after first run
+    && \in_array($process->run(), [0, 1], true)
+) {
```

Comment:
little issue with `ecs` need to run it multiple times to apply all rulesets.

Reply:
This should not happen. It's probably wrong race-condition of array rules priority. Could you make an issue report on Symplify? I'll check it

Reply:
Seems to be fixed know. Thx!